numpy/_pytesttester.py

```diff
@@ -95,6 +96,10 @@ def __call__(self, label='fast', verbose=1, extra_argv=None,
         durations : int, optional
             If < 0, do nothing, If 0, report time of all tests, if > 0,
             report the time of the slowest `timer` tests. Default is -1.
+        capture : {None, 'sys', 'fd', 'suppress'}, optional
+            Sets the stdout/stderr capturing level used by pytest,
+            according to https://docs.pytest.org/en/latest/capture.html#setting-capturing-methods-or-disabling-capturing
+            If `None`, use the default (usually `'fd'`).
```

Comment:
Need `.. versionadded:: 1.17.0` with a preceding blank line. Grep for examples.

Reply:
Done